include/af/defines.h

```diff
@@ -254,7 +254,12 @@ typedef enum {
     ///
     /// Out of bound values are symmetric over the edge
     ///
-    AF_PAD_SYM
+    AF_PAD_SYM,
+
+    ///
+    /// Out of bound values are clamped to the edge
+    ///
+    AF_PAD_CLAMP_TO_EDGE,
```

Comment:
Adding an enum will not affect the ABI. Its unfortunate we have to add this to the header but its necessary to avoid deviating from the master branch.